include/bluetooth/conn.h

```diff
@@ -299,6 +299,9 @@ struct bt_conn_le_info {
 #endif /* defined(CONFIG_BT_USER_DATA_LEN_UPDATE) */
 };
 
+/* Multiply bt 1.25 to get MS */
+#define BT_INTERVAL_TO_MS(interval) ((interval) * 5 / 4)
```

Comment:
To follow naming scheme in this file

```suggestion
#define BT_CONN_INTERVAL_TO_MS(interval) ((interval) * 5 / 4)
```

Reply:
@jhedberg @ppryga 

We have `BT_GAP_PER_ADV_INTERVAL_TO_MS` in `gap.h`. Should this be placed next to that, or is it better to keep here in `conn.h`?